warehouse/accounts/views.py

```diff
@@ -1550,46 +1550,50 @@ def delete_pending_oidc_publisher(self):
 
         form = DeletePublisherForm(self.request.POST)
 
-        if form.validate():
-            pending_publisher = self.request.db.query(PendingOIDCPublisher).get(
-                form.publisher_id.data
+        if not form.validate():
+            self.request.session.flash(
+                "Invalid publisher ID",
+                queue="error",
             )
+            return self.default_response
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
**praise:** ‏love early exits!